File: backend/BookstoreTamAn/src/main/java/com/minhnghia/datn/BookstoreTamAn/model/Author.java
package com.minhnghia.datn.BookstoreTamAn.model;

import jakarta.persistence.*;
import lombok.*;

import java.util.Date;

@Entity
@Table(name = "author")
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Author extends BaseEntity{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String name;
    private String bio;
    private String imageUrl;
    private boolean active;
    private String createdBy;
    private String modifyBy;
}

File: frontend/frontend/src/app/models/author.ts

export class Author{
    id : 0;
    name: string;
    bio : string;
    imageUrl : string;
    active : boolean;

    constructor(){
        this.id = 0;
        this.name = "";
        this.bio = "";
        this.imageUrl = "";
        this.active = true;
    }
}

export interface IAuthorList{
    id : 0;
    name: string;
    bio : string;
    imageUrl : string;
    active : boolean;
}